converter.php

```diff
@@ -93,7 +101,7 @@ function getUserIpAddr()
     ((isset($_POST['xoff']) && $_POST['xoff'] !== '') ? ($_POST['xoff']) : $bgOffsets[$bg][2]) . '" y="' .
     ((isset($_POST['yoff']) && $_POST['xoff'] !== '') ? ($_POST['yoff']) : $bgOffsets[$bg][3]) . '" width="' .
     ((isset($_POST['xsoff']) && $_POST['xoff'] !== '') ? ($_POST['xsoff']) : $bgOffsets[$bg][4]) . '" height="' .
-    ((isset($_POST['ysoff']) && $_POST['xoff'] !== '') ? ($_POST['ysoff']) : $bgOffsets[$bg][5]) . '" href="' . $bg . '" />
+    ((isset($_POST['ysoff']) && $_POST['xoff'] !== '') ? ($_POST['ysoff']) : $bgOffsets[$bg][5]) . '" href="/topo/' . $bg . '" />
             </pattern>
```

Comment:
why did only one of the 4 backgrounds got relocated?

Reply:
Not sure on this comment. As part of the file restructure the images for the backgrounds were moved into the /topo folder so the path had to reflect that to find them.